docs/ci-cd/generate_metrics.sh

```
@@ -0,0 +1,41 @@
+        local commit_msg=$(gh api repos/:owner/:repo/commits/$full_sha --jq '.commit.message' 2>/dev/null | head -n1 | tr -d '\n\r' | tr '|' ',' || echo "N/A")
+        
+        local base_data=$(echo "$run_data" | cut -d'|' -f1,2,4,5,6)
+        echo "${base_data}|${commit_msg}"
+    done
+}
+
+create_table() {
+    echo "| Run # | Commit SHA | Status | Start → End (min) | Deployed? | Notes |" >> metrics-raw.md
+    echo "| --- | --- | --- | --- | --- | --- |" >> metrics-raw.md
+    
+    fetch_metrics | while IFS='|' read -r run_id sha status conclusion duration commit_msg; do
```

Shellcheck (0.11.0)

[warning] status appears unused. Verify use (or export if used externally).

(SC2034)